tests/reactive_fluid_transport_zero_solubility/screen-output

```diff
@@ -19,8 +19,8 @@ Number of degrees of freedom: 74 (18+7+18+4+9+9+9)
 
    Postprocessing:
      Compositions min/max/mass:   0/0/0 // 0.01/0.01/4e+04
-     Melt fraction min/total/max: 0.01, 4e+04, 0.01
-     RMS, max velocity:           8.75e-19 m/year, 1.5e-18 m/year
+     Melt fraction min/total/max: 0, 0, 0
```

Comment:
Is this correct? Why is the melt fraction zero if it was 1% before?
This does not seem right to me.